crates/oxc_semantic/src/builder.rs

```diff
@@ -31,6 +31,16 @@ use crate::{
     Semantic,
 };
 
+macro_rules! cfg {
```

Comment:
Rust analyzer breaks with macros, maybe `self.cfg.map(|cfg| {}`?

Reply:
With mapping, we can't have a single point for cfg use and it is much harder to disable it all together with flags. Maybe we can bikeshed it a bit?